src/nrepl/jack-in.ts

```diff
@@ -265,6 +260,12 @@ export async function jackIn(connectSequence: ReplConnectSequence, cb?: () => un
     try {
       projectConnectSequence = await getProjectConnectSequence();
     } catch (e) {
+      outputWindow.appendLine(`; ${e}\n; Aborting jack-in.`);
+      // TODO: Figure out why this is not shown to the user.
+      void vscode.window.showErrorMessage(e, 'OK');
```

Comment:
I wonder if awaiting this would cause it to be shown to the user.

Reply:
It doesn't. I've tried that and some 100 other things. =)